src/content/docs/plugin/file-system.mdx

```diff
@@ -84,6 +84,14 @@ Install the fs plugin to get started.
 
 The fs plugin is available in both JavaScript and Rust.
 
+:::note[Rust Exposure]
```

Comment:
on the next release we'll include a few APIs for Android support https://github.com/tauri-apps/plugins-workspace/pull/1658
for now we have fs open(), read() and read_to_string() iirc - but we'll add more
this is important to resolve content URIs (returned by the dialog open() and save() APIs for instance) and resources on Android

Reply:
Added a mention that there is some exposure for mobile.